src/List/Selection.elm

```diff
@@ -167,3 +168,44 @@ mapSelected mappers (Selection selected items) =
             )
             items
         )
+
+
+{-| Filter all items where predicate evaluates to false, preserving unfiltered
+selected item.
+
+    fromList [1, 2, 3]
```

Comment:
Having one of these examples is fine; four is starting to smell like it should be in an actual test instead of relying on verify-examples. Would you mind redoing this to show the most important thing, and only that?